### ISSUE TITLE
UX: Add Keyboard shortcut: next AI feature / skip AI feature

### ISSUE THREAD

**Issue by amenk** [facebookincubator/RapiD#22]
Currently approving the AI roads still requires mouse-work. I would be more efficient if there is a key to select the next nearby feature, or a mode, where the next nearby feature is selected once A or D is pressed. But we need a key as well for skipping, if unsure in the moment.

**Comment by Bonkles** [comment]
The team is definitely aware that doing lots of road adding/deleting is a bit mouse intensive. That said, this feature poses some interesting challenges! Some questions we would need to think through: 

1) How to handle the case where the user adds the feature, but wants to make a change before proceeding? they probably wouldn't want to always auto-select the next feature after hitting 'A'. 
2) In what order should we do the 'next/prev' selections? 
3) Could this feature be implemented in a different way, such as selecting multiple AI features at once and batch-accept or batch-deleting them? 

Will take this to the team and see what shakes out.

**Comment by jjiglesiasg** [comment]
I agree that perhaps a "next feature" will be a fine addittion, is not indeed to much mouse work, because the roads added usually requires edition, in particular in curvy roads in the connection areas with other roads.

The issue is to been looking around the map to find the closest "feaute" it will be very helpfull to have that "auto" posibility... Thanks

**Comment by cbeddow** [comment]
@Bonkles hotkeys for accept/reject are in place now right? can close this?

**Comment by bhousel** [comment]
Let's leave it open for now.. We can think of ways we could make the Accept/Reject a little more efficient.
